File: core/src/main/java/com/hgianastasio/biblioulbrav2/core/user/models/RenewLoansResponse.java
package com.hgianastasio.biblioulbrav2.core.user.models;

/**
 * Created by heitorgianastasio on 4/25/17.
 */

public class RenewLoansResponse {
    private boolean status;
    private String message;

    public RenewLoansResponse() {
    }

    public RenewLoansResponse(boolean status, String message) {
        this.status = status;
        this.message = message;
    }

    public boolean getStatus() {
        return status;
    }

    public void setStatus(boolean status) {
        this.status = status;
    }

    public String getMessage() {
        return message;
    }

    public void setMessage(String message) {
        this.message = message;
    }
}

File: core/src/main/java/com/hgianastasio/biblioulbrav2/core/user/models/RenewLoansResponse.kt
package com.hgianastasio.biblioulbrav2.core.user.models

/**
 * Created by heitorgianastasio on 4/25/17.
 */
class RenewLoansResponse {
    var status = false
    var message: String? = null

    constructor() {}
    constructor(status: Boolean, message: String?) {
        this.status = status
        this.message = message
    }

}